decidim-proposals/app/packs/stylesheets/decidim/proposals/proposals.scss

```diff
@@ -0,0 +1,2 @@
+/* stylelint-disable */
```

Comment:
do we really need this file ?

Reply:
Not anymore. It was a workaround when this branch and proposals were in parallel: the styles were introduced in the other branch, nevertheless this entrypoint required a file to compile webpack.

I just removed it.